ampeg.testing/src/test/java/HeritageManualsPage.java

```diff
@@ -0,0 +1,17 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HeritageManualsPage extends PageObject {
+
+	public HeritageManualsPage(WebDriver driver) {
+		super(driver);
+	}
+
+	public PDFpage openFirstPDF() {
+		find(By.xpath("//table/tbody/tr[1]/td[2]"))
```

Comment:
What if the table structure changes? How will you know if this is still the first PDF?

Reply:
The top left cell should not be empty otherwise the table is poorly formated.
I changed the xpath to be more specific